src/stories/pages/Login/Login.js

```diff
@@ -24,47 +23,30 @@ import { Button } from '../../components/Button/Button';
 export const Login = () => {
   const [email, setEmail] = useState(null);
   const [password, setPassword] = useState(null);
+  // const [error, setError] = useState('');
+  const [loading, setLoading] = useState(false);
+  const { login } = useContext(AuthContext);
   const history = useHistory();
 
-  /**
-   * Logs in with the current values from email and password,
-   *   navigates to home if succesful login.
-   */
   const handleLogin = () => {
-    fire
-      .auth()
-      .signInWithEmailAndPassword(email, password)
-      .catch((err) => {
-        switch (err.code) {
-        case 'auth/invalid-email':
-        case 'auth/user-disabled':
-        case 'auth/user-not-found':
```

Comment:
I wonder if at some point we should match this to HTTP error codes.

Reply:
Yeah we should. Once we have an error component, we can send a custom message based on returned error code.